docs/rules/no-mutating-props.md

```diff
@@ -88,7 +120,45 @@ This rule reports mutation of component props.
 
 ## :wrench: Options
 
-Nothing.
+```json
+{
+  "vue/no-mutating-props": ["error", {
+    "propProps": true
```

Comment:
Did you see the comment below regarding option names? Could you please comment what you think about those names?
https://github.com/vuejs/eslint-plugin-vue/issues/1371#issuecomment-1064620364

Reply:
Thank you for your reply! I think "shallowOnly" is more appropriate. It has the same naming style with vue3.